components/EmptyTokensText.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react'
 import { StyleSheet, View } from 'react-native'
-import { Subheading } from 'react-native-paper'
 
 import theme from '../lib/defaultTheme'
 
+import { Headline, BodyText } from './typography'
 import Spacer from './Spacer'
 
 const UI_STRINGS = {
-  heading: `You don't have a secret set up yet.`,
-  description:
-    'Add one by scanning or uploading a QR code, or even enter the details manually.',
+  heading: 'No Secrets',
+  description: 'Add a new secret and it will appear here.',
 }
 
 export default function EmptyTokensText() {
   return (
     <View style={styles.description}>
-      <Subheading>{UI_STRINGS.heading}</Subheading>
+      <Headline level="5" color={theme.colors.grey}>
+        {UI_STRINGS.heading}
+      </Headline>
       <Spacer size={2} />
-      <Subheading>{UI_STRINGS.description}</Subheading>
+      <BodyText color={theme.colors.grey}>{UI_STRINGS.description}</BodyText>
```

Comment:
because all the text except the one on dark background has this color, shall we not use the textColor from the theme (or whatever it's called). or maybe, even make it the default on our custom typography in the same way as it's done for the paper typography

Reply:
The `text` color from the default theme in `react-native-paper` points to `#000000`.  https://github.com/callstack/react-native-paper/blob/main/src/styles/DefaultTheme.tsx#L15. I'll make it the default for custom typography.